lib/rules/web/admin_console/4.2/deployment.xyaml

```diff
@@ -30,3 +30,31 @@ search_image:
   action: clear_input_value
   action: set_input_value
   action: click_search_button
+deploy_search_nonexisted_image:
```

Comment:
Since we didn't deploy app using this step, so I would call it `search_nonexisted_image_and_check_message`